pkg/storage/prunable/prunable.go

```diff
@@ -142,16 +142,16 @@ func (p *Prunable) Flush() {
 	}
 }
 
-func (p *Prunable) Rollback(targetEpoch iotago.EpochIndex, pruningRange [2]iotago.SlotIndex) error {
-	if err := p.prunableSlotStore.PruneSlots(targetEpoch, pruningRange); err != nil {
-		return ierrors.Wrapf(err, "failed to prune slots in range [%d, %d] from target epoch %d", pruningRange[0], pruningRange[1], targetEpoch)
+func (p *Prunable) Rollback(targetEpoch iotago.EpochIndex, startPruneRange iotago.SlotIndex, endPruneRange iotago.SlotIndex) error {
+	if err := p.prunableSlotStore.PruneSlots(targetEpoch, startPruneRange, endPruneRange); err != nil {
+		return ierrors.Wrapf(err, "failed to prune slots in range [%d, %d] from target epoch %d", startPruneRange, endPruneRange, targetEpoch)
 	}
 
-	if err := p.rollbackCommitteesCandidates(targetEpoch, pruningRange[0]-1); err != nil {
+	if err := p.rollbackCommitteesCandidates(targetEpoch, startPruneRange); err != nil {
```

Comment:
You removed the original `-1` from the startPruneRange. Was it intended?

Reply:
I fixed a bug in the condition as it was `candidacySlot < targetSlot` which lead to the wrong slots being deleted. 

And yes, it was intended because I changed it to be inclusive `candidacySlot >= deletionStartSlot`. So we delete everything starting and including `deletionStartSlot`